sunkit_image/conftest.py

```diff
@@ -7,7 +7,6 @@
 
 import sunpy.tests.helpers
 from sunpy.tests.hash import HASH_LIBRARY_NAME
-from sunpy.tests.helpers import new_hash_library, generate_figure_webpage
```

Comment:
Note: We will probably need this when we setup figure tests. 

I might setup the env for them and then we can add some in this PR.

Reply:
Sorry I did not get you. Should I keep this line unchanged??

Reply:
I will probably open a PR adding the code/files needed for the figure tests so we can look a figure test or two in this PR as well.

Reply:
The deletion of this line is causing tox tests to fail, I guess. Because here import of ```hash_library``` is deleted but it is used in the function ```pytest_unconfigure``` which is throwing an error. Have a look at the error on this [page](https://dev.azure.com/sunpy/sunkit-image/_build/results?buildId=2069). The pytest tests pass but the same error pops up.
```bash
(base) vatsalya@vatsalya-UX430UNR:~/MEGA/sunkit-image$ conda activate sunkit-dev
(sunkit-dev) vatsalya@vatsalya-UX430UNR:~/MEGA/sunkit-image$ pytest -v
Internet access disabled
================================================================================ test session starts =================================================================================
platform linux -- Python 3.7.3, pytest-4.3.1, py-1.8.0, pluggy-0.9.0 -- /home/vatsalya/anaconda3/envs/sunkit-dev/bin/python
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/home/vatsalya/MEGA/sunkit-image/.hypothesis/examples')
rootdir: /home/vatsalya/MEGA/sunkit-image, inifile: setup.cfg
plugins: remotedata-0.3.1, openfiles-0.3.2, mock-1.10.4, doctestplus-0.3.0, cov-2.7.1, arraydiff-0.3, hypothesis-4.23.6
collecting ... /home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/glymur/config.py:152: UserWarning: Neither the openjp2 nor the openjpeg library could be loaded.  
  warnings.warn(msg)
/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/glymur/config.py:152: UserWarning: Neither the openjp2 nor the openjpeg library could be loaded.  
  warnings.warn(msg)
collected 15 items                                                                                                                                                                   

sunkit_image/tests/test_offlimb_enhance.py::test_normalizing_radial_gradient_filter PASSED                                                                                     [  6%]
sunkit_image/tests/test_offlimb_enhance.py::test_fourier_normalizing_radial_gradient_filter PASSED                                                                             [ 13%]
sunkit_image/utils/noise.py::sunkit_image.utils.noise.NoiseLevelEstimation.__init__ PASSED                                                                                     [ 20%]
sunkit_image/utils/tests/test_utils.py::test_equally_spaced_bins PASSED                                                                                                        [ 26%]
sunkit_image/utils/tests/test_utils.py::test_bin_edge_summary PASSED                                                                                                           [ 33%]
sunkit_image/utils/tests/test_utils.py::test_find_pixel_radii SKIPPED                                                                                                          [ 40%]
sunkit_image/utils/tests/test_utils.py::test_get_radial_intensity_summary PASSED                                                                                               [ 46%]
docs/format.rst PASSED                                                                                                                                                         [ 53%]
docs/index.rst PASSED                                                                                                                                                          [ 60%]
docs/code_ref/index.rst PASSED                                                                                                                                                 [ 66%]
docs/code_ref/offlimb_enhance.rst PASSED                                                                                                                                       [ 73%]
docs/code_ref/sunkit_image.rst PASSED                                                                                                                                          [ 80%]
docs/code_ref/utils.rst PASSED                                                                                                                                                 [ 86%]
docs/whatsnew/changelog.rst PASSED                                                                                                                                             [ 93%]
docs/whatsnew/index.rst PASSED                                                                                                                                                 [100%]

======================================================================== 14 passed, 1 skipped in 4.70 seconds ========================================================================
Traceback (most recent call last):
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/bin/pytest", line 11, in <module>
    sys.exit(main())
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 79, in main
    return config.hook.pytest_cmdline_main(config=config)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/hooks.py", line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/manager.py", line 68, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/manager.py", line 62, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/_pytest/main.py", line 242, in pytest_cmdline_main
    return wrap_session(config, _main)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/_pytest/main.py", line 237, in wrap_session
    config._ensure_unconfigure()
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/_pytest/config/__init__.py", line 643, in _ensure_unconfigure
    self.hook.pytest_unconfigure(config=self)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/hooks.py", line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/manager.py", line 68, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/manager.py", line 62, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/home/vatsalya/anaconda3/envs/sunkit-dev/lib/python3.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/home/vatsalya/MEGA/sunkit-image/sunkit_image/conftest.py", line 48, in pytest_unconfigure
    if len(new_hash_library) > 0:
NameError: name 'new_hash_library' is not defined
```